models/employee.py

```diff
@@ -30,15 +30,41 @@ def _forget_results():
 
 class Employee(ndb.Model, Pagination):
     """Models an Employee."""
-    department = ndb.StringProperty(indexed=False)
+    department = ndb.StringProperty(indexed=True)
     first_name = ndb.StringProperty(indexed=False)
     last_name = ndb.StringProperty(indexed=False)
-    meta_department = ndb.StringProperty()
+    meta_department = ndb.StringProperty(indexed=True)
     photo_url = ndb.TextProperty()
     terminated = ndb.BooleanProperty(default=False)
     timestamp = ndb.DateTimeProperty(auto_now_add=True)
     user = ndb.UserProperty()
     username = ndb.StringProperty()
+    office = ndb.StringProperty(indexed=True)
+
+    @classmethod
+    def refresh_indexes(cls):
```

Comment:
My plan is to run this method only once after deployment. Since I have added new index and the only way to make it work according to the docs is to refresh the table again.

Reply:
Could you reference those docs please? This method is basically recreating all datastore entries, and I don't think that's necessary or advisable to do.

[The documentation I found](https://cloud.google.com/appengine/docs/standard/python/tools/uploadinganapp#updating_indexes) mentions two ways of dealing with new indexes: Either uploading just `index.yaml` and monitoring index creation progress, or uploading the new version and not routing traffic to it until indexes have been created. Either of those sound fine to me. I think you can remove this method.